plugins/PrivacyManager/lang/en.json

```diff
@@ -83,6 +83,9 @@
         "DoNotTrack_SupportDNTPreference": "Support Do Not Track preference",
         "EstimatedDBSizeAfterPurge": "Estimated database size after purge",
         "EstimatedSpaceSaved": "Estimated space saved",
+        "ForceCookielessTracking": "Force tracking without cookies",
+        "ForceCookielessTrackingDescription": "Enabling this option will automatically update %1$s, so it contains some additional code to ensure all trackers won't use cookies. Additionally Matomo will ignore all tracking cookies on server side.",
+        "ForceCookielessTrackingDescriptionNotWritable": "Your %1$s is not writable. Therefor it can't be updated, to ensure cookies won't be used. Please make the file writable or ensure to call disableCookies manually for all javascript trackers. Otherwise Matomo will only ignore all tracking cookies on server side.",
```

Comment:
Could maybe say something like:

```
The JS tracker file "piwik.js" is not writable which is required for this feature to fully work. When the JS tracker file is not writable, then the cookies will be only ignored on the server side but cookies might be still set in the browser. To solve this issue you or a system administrator need to change the file permissions so your webserver can change this file.
```

Ideally there was a small FAQ explaining how this maybe can be done to give people some idea of what needs to happen. It be bit difficult because of the different operating systems etc. Or could we maybe show a command here that they can run? Maybe we have this already in Matomo from other directory write checks? Then we could add at the end: `For example you might be able to fix this by executing the command "chmod g+w /path/to/matomo/matomo/js"` but I suppose we can't really know if the user, the group needs to add write access (we wouldn't want to suggest give write access for `all`). 

Maybe later we could also have a feature for Matomo to try to give write access but not too important for now and often it wouldn't work anyway.